js/apps/admin-ui/cypress/support/pages/admin-ui/manage/AttributesTab.ts

```diff
@@ -1,11 +1,25 @@
 export default class AttributesTab {
   #saveAttributeBtn = "save-attributes";
-  #addAttributeBtn = "attributes-add-row";
   #attributesTab = "attributes";
-  #keyInput = "attributes-key";
-  #valueInput = "attributes-value";
-  #removeBtn = "attributes-remove";
   #emptyState = "attributes-empty-state";
+  #addAttributeBtn: string;
+  #keyInput: string;
+  #valueInput: string;
+  #removeBtn: string;
+
+  constructor(context: string) {
```

Comment:
As mentioned before, since there is only one parameter that is valid we can type this a bit better.
```suggestion
  constructor(context?: "user") {
```

However, perhaps it would make more sense to make this a boolean instead?

```suggestion
  constructor(isForUser = false) {
```